docs/GDI-SOP_github-introduction-for-maintainers.md

```diff
@@ -0,0 +1,193 @@
+- **Issue** (_green_): A way to track tasks, improvements, or bugs in the repository.
+
+![Basic layout of GitHub](images/GDI-SOP_github-introduction-for-maintainers_basic-layout.png)
+
+We will elaborate on these key concepts further in the sections below.
+
+### Example of GitHub usage
+See the diagram below for a simple example of how changes are propagated through the branches of this repository.
+
+From left to right:
+- The GDI SOP repository (`main` branch) was created.
+- New changes (e.g., documentation files) were added.
+- A new branch (`dev`) was created to hold 'new features' until they were ready to be part of the `main` branch.
+- A new SOP was scheduled, so another branch, `new-SOP`, was created.
+- After adding and reviewing the new SOP, it was merged into the `dev` branch.
+- More work was done on `dev` (e.g., adding new documents, SOPs, modifying content). Keep in mind that so far, the `main` branch remained unaltered.
+- Eventually, when the new changes in `dev` (including the "new SOP") were ready, `dev` was merged into `main`.
+
+````mermaid
+gitGraph
+    commit id: "Initial project"
+    commit id: "New changes"
+    branch dev
+    checkout dev
+    commit id: "Create 'dev' branch"
+    commit id: "More changes"
+    branch new-SOP
+    checkout new-SOP
+    commit id: "Create new branch"
+    commit id: "Add new SOP"
+    commit id: "Revise new SOP"
+    checkout dev
+    merge new-SOP id: "Merge new SOP into dev"
+    commit id: "Continue work on dev"
+    checkout main
+    merge dev id: "Merge dev into main"
+    commit id: " "
+````
+
+## Understanding the Repository
+
+This repository has a specific folder and file structure, including:
+- **`/docs/`**: Contains documentation files, including this guide.
+- **`/sops/`**: Contains the Standard Operating Procedures (SOPs).
+- **`/scripts/`**: Contains Python scripts used in workflows to automate certain processes. These scripts are called from elsewhere, either manually or automatically, to ease some tasks.
+- **`/.github/`**: Contains bits that you could consider part of the "settings" of the repository, like Pull Request and Issue templates, as well as workflows.
+- **`README.md`**: The main landing page for the repository, explaining its purpose. There may be multiple other ``README.md`` files in other folders, which are rendered by GitHub as soon as you enter a directory.
+
+Refer to the [**GDI-SOP charter**](../docs/GDI-SOP_charter.md) for more information on the structure and purpose of this repository.
+
+Refer to the [**SOP accessioning guide**](../docs/GDI-SOP_sop-accessioning.md) for more details on how files in this repo are named and organised.
+
+## Issues
+
+Issues are used to track tasks, requests, or bugs. When you (or anyone) creates an issue in the repository, it automatically gets added to the [**ZenHub board**](https://app.zenhub.com/workspaces/t43-gdi-sops-667c1c5532726a00b93d51e4/board) used by GDI for project management.
```

Comment:
I am not sure about this... but you wrote... When you (or anyone) create**s** an issue ... it should be "creates" or "create", as it refers probably to "you",

Reply:
Good point, however, the addition of "(or anyone)" modifies the sentence to refer to a generalised, indefinite subject (i.e., either "you" or "anyone"), which is typically treated as singular. This shifts the verb to "creates." - If there's a native among us please let us know, but ChatGPT seems to agree on this 😂

Reply:
I am not a native speaker, and that's why I was not so sure whether my comment was correct or not! But ChatGPT has saved us, I guess!! :-)